### ISSUE TITLE
Inspect can't find PropertyGrid's highlighted dropdown list in the accessibility tree

### ISSUE THREAD

**Issue by SergeySmirnov-Akvelon** [dotnet/winforms#4876]
**.NET Core Version:**
- 6.0.100-preview.3.21202.5

**Have you experienced this same bug with .NET Framework?:**
- Yes

**Minimal repro:**
1. Create an app with a PropertyGrid and button.
2. Set button as "SelectedObject" property of PropertyGrid
3. Run the app
4. Run  the Inspect
5. Expand the "AccessibleRole" dropdown list and highlight the drop-down list using the Inspect

**Actual behavior:**
The highlighted drop-down list is not displayed in the accessibility tree.
![image](https://user-images.githubusercontent.com/23376742/117441815-9d7d4d80-af3e-11eb-8e62-e10c914b4a2a.png)

**Expected behavior:**
The highlighted drop-down list should be displayed in the accessibility tree.

**Comment by vladimir-krestov** [comment]
Looks like there is an incorrect accessibility tree. "Available values" list and DropDownHolder are different components.
DropDownHolder includes the values list. 
- So the first way is to add a new "DropDownHolder" layer (red in the pic), like this:
![image](https://user-images.githubusercontent.com/49272759/141472510-82eea00f-c203-400e-9bd4-a16939932f19.png)
- The second way is to change HitTest logic (get element from mouse point) and the dropped list box Bounds, when mouse hovers DropDownHolder we may return GridViewListBoxAO instead of DropDownHolderAO, like that:
![AR0X7imhBk](https://user-images.githubusercontent.com/49272759/141612743-77653d33-0e24-498c-82d9-74a7bbfc5677.gif)

@dreddy-work, which way looks more acceptable?

**Comment by Tanya-Solyanik** [comment]
Could you please elaborate on these options? 
>So the first way is to add a new "DropDownHolder" layer (red in the pic), like this:

Where are you suggesting to add DroppDownHolder to? Are you suggesting to return it's accessibility object from the navigation APIs?

The second tree looks cleaner, and might be preferable to user experience. But I would think that navigation interfces would need to change as well to ensure parent-child link between the propertygrid entry and the list of available items.

What would you prefer?

**Comment by vladimir-krestov** [comment]
Actually, GridViewListBox is a component of DropDownHolder, so the first way is to change `FragmentNavigate` logic of `DropDownHolderAccessibleObject` and `GridViewListBoxAccessibleObject` to configure parent-child relationships. Thereby I will change the accessibility tree. In my opinion, this is right way, because actually we see 2 components DropDownHolder and ListBox inside, and the accessibility tree should reflect the real status. The second way deceives users, because accessible object will reflect not real control.
But, a Narrator user won't see the changes, because Narrator focuses on a selected item and doesn't announces the list and drop-down holder. The "Scan" mode won't help to get them.
So we won't change users experience, and my choice is the first way (add a DropDownHolder in the tree via FragmentNavigate)

**Comment by Tanya-Solyanik** [comment]
If we already have `DropDownHolderAccessibleObject` and `GridViewListBoxAccessibleObject`, it makes sense to define a parent-child relationship between them, the first option sounds good.

**Comment by Amy-Li03** [comment]
Verified this issue on .NET 7.0.100.preview.5.22227.7, issue was fixed. There is "Drop down control holder" panel node in "AccrssibleRole" tree item and the highlighted drop-down list is displayed in the accessibility tree.
![image](https://user-images.githubusercontent.com/62929087/165667708-177f7b62-bbea-4dce-a026-e451f37c5fbb.png)
